Files/UserControls/Widgets/Bundles.xaml

```diff
@@ -0,0 +1,362 @@
+													<FontIcon FontFamily="{StaticResource FluentUIGlyphs}" Glyph="&#xe9ed;"/>
+												</MenuFlyoutItem.Icon>
+											</MenuFlyoutItem>
+											<MenuFlyoutItem
+												Text="Rename Bundle"
+												Command="{Binding RenameBundleCommand, Mode=OneWay}">
+												<MenuFlyoutItem.Icon>
+													<FontIcon FontFamily="{StaticResource FluentUIGlyphs}" Glyph="&#xeb3f;"/>
+												</MenuFlyoutItem.Icon>
+											</MenuFlyoutItem>
+										</MenuFlyout>
+									</Button.Flyout>
+								</Button>
+							</Grid>
+
+							<!-- List of Bundle Items -->
+							<GridView
+								SelectionMode="None"
+								VerticalAlignment="Stretch"
+								HorizontalAlignment="Stretch"
+								ItemsSource="{Binding Contents, Mode=OneWay}">
+								<GridView.ItemTemplate>
+									<!-- Bundle Item design -->
+									<DataTemplate>
+										<Grid>
+											<Button
+												Padding="0"
+												HorizontalAlignment="Stretch"
+												VerticalAlignment="Stretch"
+												HorizontalContentAlignment="Center"
+												VerticalContentAlignment="Center"
+												BorderThickness="1"
+												Width="120"
+												Height="35"
+												Command="{Binding OpenItemCommand, Mode=OneWay}">
+
+												<StackPanel Orientation="Horizontal" Spacing="3">
+													<!-- Icon -->
+													<Image
+														Width="20"
+														Height="20"
+														Source="{Binding Icon, Mode=OneWay}"/>
+													<TextBlock
+														Text="{Binding Name, Mode=OneWay}"
+														FontSize="14"
+														TextWrapping="Wrap"
+														VerticalAlignment="Center"
+														HorizontalAlignment="Center"/>
+												</StackPanel>
+
+												<Button.ContextFlyout>
+													<MenuFlyout>
+														<MenuFlyoutItem
+															Text="Remove Item"
+															Command="{Binding RemoveItemCommand, Mode=OneWay}">
+															<MenuFlyoutItem.Icon>
+																<FontIcon FontFamily="{StaticResource FluentUIGlyphs}" Glyph="&#xe9ed;"/>
+															</MenuFlyoutItem.Icon>
+														</MenuFlyoutItem>
+													</MenuFlyout>
+												</Button.ContextFlyout>
+
+												<ToolTipService.ToolTip>
+													<ToolTip Content="{Binding Path, Mode=OneWay}"/>
+												</ToolTipService.ToolTip>
+											</Button>
+										</Grid>
+									</DataTemplate>
+								</GridView.ItemTemplate>
+							</GridView>
+
+							<!-- No Bundle contents text info -->
+							<Grid VerticalAlignment="Center" HorizontalAlignment="Center" RowSpacing="3" Visibility="{Binding NoBundleContentsTextVisibility, Mode=OneWay}">
+								<Grid.RowDefinitions>
+									<RowDefinition/>
+									<RowDefinition/>
+								</Grid.RowDefinitions>
+
+								<FontIcon
+									FontFamily="{StaticResource FluentUIGlyphs}"
+									Glyph=""/>
+								<TextBlock
+									Text="Drag and drop any folder,&#10;file or program here to&#10;quickly access it"
```

Comment:
```suggestion
									Text="Drag and drop any item here to quickly access it"
```

Reply:
"any item" here is not very precise, user might drop something that we didn't expect